test/parallel/proofOfStake.3rdPartyRewards.activation.test.ts

```diff
@@ -136,6 +127,18 @@ describe("Proof of stake tests", () => {
         ),
       ),
     );
+    const liqId = await getLiquidityAssetId(GASP_ASSET_ID, newToken);
+    await Sudo.batchAsSudoFinalized(
+      Sudo.sudoAs(
+        testUser3,
+        Market.mintLiquidity(
+          liqId,
+          GASP_ASSET_ID,
+          Assets.DEFAULT_AMOUNT,
+          Assets.DEFAULT_AMOUNT.muln(2),
+        ),
+      ),
+    );
```

Comment:
can you do this before, so the mintLiquidity happen in the same batch?  or perhaps the pool is created in the same batch?